ui/main/src/assets/js/opfab.js

```
@@ -150,33 +139,31 @@ class QuillEditor extends HTMLElement {
     }
 
     init() {
-
         const Link = Quill.import('formats/link');
 
         class CustomLinkSanitizer extends Link {
             static sanitize(url) {
+                const sanitizedUrl = super.sanitize(url);
```

Biome (1.9.4)

[error] Using super in a static context can be confusing.

super refers to a parent class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)